.github/scripts/bash/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+BASH_PATH=${BASH_PATH:-~/$TARGET/bash}
+BASH_BUILD_PATH=$BUILD_PATH/bash
+BASH_SOURCE_PATH=$SOURCE_PATH/bash
+
+PATH="$TOOLCHAIN_PATH/bin:$TOOLCHAIN_PATH/lib/gcc/$TARGET/:$TOOLCHAIN_PATH/lib/gcc/$TARGET/$GCC_VERSION/:$PATH"
```

Comment:
JFYI: These two changes to the environment should teoretically be part of the `config.sh`. Historically, there was an issue with bootstrapping of the toolchain if it was so but it might no longer be needed or even an idication of an issue with our target configuration. I am planning to take a look at it again , one day...

Reply:
Thanks, it is not needed anymore.